lib/highLevelConsumer.js

```diff
@@ -523,11 +518,13 @@ HighLevelConsumer.prototype.offsetRequest = function (payloads, cb) {
  * @param {Client~failedToRegisterConsumerCallback} cb A function to call the consumer has been registered
  */
 HighLevelConsumer.prototype.registerConsumer = function (cb) {
-    this.client.zk.registerConsumer(this.options.groupId, this.id, this.payloads, function (err) {
+    var zk = this.client.zk;
+    var groupId = this.options.groupId;
+    this.client.zk.registerConsumer(groupId, this.id, this.payloads, function (err) {
```

Comment:
Doesn't look like `zk` is actually being used outside the context of the `client` so we could delete [line 12 and 13](https://github.com/SOHU-Co/kafka-node/blob/master/lib/highLevelConsumer.js#L12)